torch/_tensor_docs.py

```diff
@@ -1983,6 +1983,20 @@ def add_docstr_all(method, docstr):
 See :func:`torch.kthvalue`
 """)
 
+add_docstr_all('linalg_eigh',
```

Comment:
Once the method variants are removed these would be, too.

Note that for these to render properly they would have needed entries in tensors.rst, too.